2023-11-24 18:36:21.970  INFO 8292 --- [restartedMain] c.a.n.client.env.SearchableProperties    : properties search order:PROPERTIES->JVM->ENV->DEFAULT_SETTING
2023-11-24 18:36:22.111  INFO 8292 --- [restartedMain] com.yc.ResfoodApp                        : Starting ResfoodApp using Java 1.8.0_372 on DESKTOP-2GHSLBQ with PID 8292 (D:\workspaceYC134\res134_cloud\res_food\target\classes started by xinye in D:\workspaceYC134\res134_cloud)
2023-11-24 18:36:22.111  INFO 8292 --- [restartedMain] com.yc.ResfoodApp                        : No active profile set, falling back to 1 default profile: "default"
2023-11-24 18:36:22.158  INFO 8292 --- [restartedMain] .e.DevToolsPropertyDefaultsPostProcessor : Devtools property defaults active! Set 'spring.devtools.add-properties' to 'false' to disable
2023-11-24 18:36:22.158  INFO 8292 --- [restartedMain] .e.DevToolsPropertyDefaultsPostProcessor : For additional web related logging consider setting the 'logging.level.web' property to 'DEBUG'
2023-11-24 18:36:23.080  INFO 8292 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Multiple Spring Data modules found, entering strict repository configuration mode!
2023-11-24 18:36:23.080  INFO 8292 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data Redis repositories in DEFAULT mode.
2023-11-24 18:36:23.095  INFO 8292 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 7 ms. Found 0 Redis repository interfaces.
2023-11-24 18:36:23.267  INFO 8292 --- [restartedMain] o.s.cloud.context.scope.GenericScope     : BeanFactory id=687256cf-2ea5-388d-9056-54b0e363ecfd
2023-11-24 18:36:23.658  INFO 8292 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 9001 (http)
2023-11-24 18:36:23.673  INFO 8292 --- [restartedMain] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2023-11-24 18:36:23.673  INFO 8292 --- [restartedMain] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.63]
2023-11-24 18:36:23.798  INFO 8292 --- [restartedMain] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2023-11-24 18:36:23.798  INFO 8292 --- [restartedMain] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 1640 ms
2023-11-24 18:36:23.939  INFO 8292 --- [restartedMain] c.a.d.s.b.a.DruidDataSourceAutoConfigure : Init DruidDataSource
2023-11-24 18:36:24.673  INFO 8292 --- [restartedMain] com.alibaba.druid.pool.DruidDataSource   : {dataSource-1} inited
2023-11-24 18:36:27.845  INFO 8292 --- [restartedMain] o.s.b.d.a.OptionalLiveReloadServer       : LiveReload server is running on port 35729
2023-11-24 18:36:28.066  INFO 8292 --- [restartedMain] o.s.b.a.e.web.EndpointLinksResolver      : Exposing 2 endpoint(s) beneath base path '/actuator'
2023-11-24 18:36:28.129  INFO 8292 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 9001 (http) with context path ''
2023-11-24 18:36:28.176  INFO 8292 --- [restartedMain] c.a.n.p.a.s.c.ClientAuthPluginManager    : [ClientAuthPluginManager] Load ClientAuthService com.alibaba.nacos.client.auth.impl.NacosClientAuthServiceImpl success.
2023-11-24 18:36:28.176  INFO 8292 --- [restartedMain] c.a.n.p.a.s.c.ClientAuthPluginManager    : [ClientAuthPluginManager] Load ClientAuthService com.alibaba.nacos.client.auth.ram.RamClientAuthServiceImpl success.
2023-11-24 18:36:29.397  INFO 8292 --- [restartedMain] c.a.c.n.registry.NacosServiceRegistry    : nacos registry, DEFAULT_GROUP resfood 192.168.186.105:9001 register finished
2023-11-24 18:36:29.652  INFO 8292 --- [restartedMain] s.a.ScheduledAnnotationBeanPostProcessor : No TaskScheduler/ScheduledExecutorService bean found for scheduled processing
2023-11-24 18:36:29.652  INFO 8292 --- [restartedMain] com.yc.ResfoodApp                        : Started ResfoodApp in 8.201 seconds (JVM running for 9.677)
2023-11-24 18:36:30.151  INFO 8292 --- [RMI TCP Connection(5)-192.168.186.105] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
2023-11-24 18:36:30.151  INFO 8292 --- [RMI TCP Connection(5)-192.168.186.105] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2023-11-24 18:36:30.151  INFO 8292 --- [RMI TCP Connection(5)-192.168.186.105] o.s.web.servlet.DispatcherServlet        : Completed initialization in 0 ms
2023-11-24 18:36:40.982  INFO 1188 --- [restartedMain] c.a.n.client.env.SearchableProperties    : properties search order:PROPERTIES->JVM->ENV->DEFAULT_SETTING
2023-11-24 18:36:41.091  INFO 1188 --- [restartedMain] com.yc.ResfoodApp                        : Starting ResfoodApp using Java 1.8.0_372 on DESKTOP-2GHSLBQ with PID 1188 (D:\workspaceYC134\res134_cloud\res_food\target\classes started by xinye in D:\workspaceYC134\res134_cloud)
2023-11-24 18:36:41.091  INFO 1188 --- [restartedMain] com.yc.ResfoodApp                        : No active profile set, falling back to 1 default profile: "default"
2023-11-24 18:36:41.122  INFO 1188 --- [restartedMain] .e.DevToolsPropertyDefaultsPostProcessor : Devtools property defaults active! Set 'spring.devtools.add-properties' to 'false' to disable
2023-11-24 18:36:41.122  INFO 1188 --- [restartedMain] .e.DevToolsPropertyDefaultsPostProcessor : For additional web related logging consider setting the 'logging.level.web' property to 'DEBUG'
2023-11-24 18:36:41.760  INFO 1188 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Multiple Spring Data modules found, entering strict repository configuration mode!
2023-11-24 18:36:41.760  INFO 1188 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data Redis repositories in DEFAULT mode.
2023-11-24 18:36:41.776  INFO 1188 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 5 ms. Found 0 Redis repository interfaces.
2023-11-24 18:36:41.920  INFO 1188 --- [restartedMain] o.s.cloud.context.scope.GenericScope     : BeanFactory id=687256cf-2ea5-388d-9056-54b0e363ecfd
2023-11-24 18:36:42.190  INFO 1188 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 9000 (http)
2023-11-24 18:36:42.190  INFO 1188 --- [restartedMain] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2023-11-24 18:36:42.190  INFO 1188 --- [restartedMain] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.63]
2023-11-24 18:36:42.304  INFO 1188 --- [restartedMain] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2023-11-24 18:36:42.304  INFO 1188 --- [restartedMain] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 1182 ms
2023-11-24 18:36:42.398  INFO 1188 --- [restartedMain] c.a.d.s.b.a.DruidDataSourceAutoConfigure : Init DruidDataSource
2023-11-24 18:36:42.839  INFO 1188 --- [restartedMain] com.alibaba.druid.pool.DruidDataSource   : {dataSource-1} inited
2023-11-24 18:36:44.938  WARN 1188 --- [restartedMain] o.s.b.d.a.OptionalLiveReloadServer       : Unable to start LiveReload server
2023-11-24 18:36:45.177  INFO 1188 --- [restartedMain] o.s.b.a.e.web.EndpointLinksResolver      : Exposing 2 endpoint(s) beneath base path '/actuator'
2023-11-24 18:36:45.237  INFO 1188 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 9000 (http) with context path ''
2023-11-24 18:36:45.275  INFO 1188 --- [restartedMain] c.a.n.p.a.s.c.ClientAuthPluginManager    : [ClientAuthPluginManager] Load ClientAuthService com.alibaba.nacos.client.auth.impl.NacosClientAuthServiceImpl success.
2023-11-24 18:36:45.275  INFO 1188 --- [restartedMain] c.a.n.p.a.s.c.ClientAuthPluginManager    : [ClientAuthPluginManager] Load ClientAuthService com.alibaba.nacos.client.auth.ram.RamClientAuthServiceImpl success.
2023-11-24 18:36:46.431  INFO 1188 --- [restartedMain] c.a.c.n.registry.NacosServiceRegistry    : nacos registry, DEFAULT_GROUP resfood 192.168.186.105:9000 register finished
2023-11-24 18:36:46.718  INFO 1188 --- [restartedMain] s.a.ScheduledAnnotationBeanPostProcessor : No TaskScheduler/ScheduledExecutorService bean found for scheduled processing
2023-11-24 18:36:46.726  INFO 1188 --- [restartedMain] com.yc.ResfoodApp                        : Started ResfoodApp in 6.259 seconds (JVM running for 6.988)
2023-11-24 18:36:47.542  INFO 1188 --- [RMI TCP Connection(2)-192.168.186.105] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
2023-11-24 18:36:47.542  INFO 1188 --- [RMI TCP Connection(2)-192.168.186.105] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2023-11-24 18:36:47.543  INFO 1188 --- [RMI TCP Connection(2)-192.168.186.105] o.s.web.servlet.DispatcherServlet        : Completed initialization in 1 ms
2023-11-24 18:37:43.635  INFO 27024 --- [main] c.a.n.client.env.SearchableProperties    : properties search order:PROPERTIES->JVM->ENV->DEFAULT_SETTING
2023-11-24 18:37:43.713  INFO 27024 --- [main] com.yc.ResorderApp                       : Starting ResorderApp using Java 1.8.0_372 on DESKTOP-2GHSLBQ with PID 27024 (D:\workspaceYC134\res134_cloud\res_order\target\classes started by xinye in D:\workspaceYC134\res134_cloud)
2023-11-24 18:37:43.713  INFO 27024 --- [main] com.yc.ResorderApp                       : No active profile set, falling back to 1 default profile: "default"
2023-11-24 18:37:44.139  INFO 27024 --- [main] o.s.cloud.context.scope.GenericScope     : BeanFactory id=0d4588b5-7c06-3826-9eb3-96630f5e29dc
2023-11-24 18:37:44.288  INFO 27024 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8000 (http)
2023-11-24 18:37:44.288  INFO 27024 --- [main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2023-11-24 18:37:44.288  INFO 27024 --- [main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.63]
2023-11-24 18:37:44.350  INFO 27024 --- [main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2023-11-24 18:37:44.350  INFO 27024 --- [main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 605 ms
2023-11-24 18:37:44.476 DEBUG 27024 --- [main] s.w.s.m.m.a.RequestMappingHandlerAdapter : ControllerAdvice beans: 0 @ModelAttribute, 0 @InitBinder, 1 RequestBodyAdvice, 1 ResponseBodyAdvice
2023-11-24 18:37:44.523 DEBUG 27024 --- [main] s.w.s.m.m.a.RequestMappingHandlerMapping : 3 mappings in 'requestMappingHandlerMapping'
2023-11-24 18:37:44.539 DEBUG 27024 --- [main] o.s.w.s.handler.SimpleUrlHandlerMapping  : Patterns [/webjars/**, /**] in 'resourceHandlerMapping'
2023-11-24 18:37:44.539 DEBUG 27024 --- [main] .m.m.a.ExceptionHandlerExceptionResolver : ControllerAdvice beans: 0 @ExceptionHandler, 1 ResponseBodyAdvice
2023-11-24 18:37:45.170  INFO 27024 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 8000 (http) with context path ''
2023-11-24 18:37:45.186  INFO 27024 --- [main] c.a.n.p.a.s.c.ClientAuthPluginManager    : [ClientAuthPluginManager] Load ClientAuthService com.alibaba.nacos.client.auth.impl.NacosClientAuthServiceImpl success.
2023-11-24 18:37:45.186  INFO 27024 --- [main] c.a.n.p.a.s.c.ClientAuthPluginManager    : [ClientAuthPluginManager] Load ClientAuthService com.alibaba.nacos.client.auth.ram.RamClientAuthServiceImpl success.
2023-11-24 18:37:46.151  INFO 27024 --- [main] c.a.c.n.registry.NacosServiceRegistry    : nacos registry, DEFAULT_GROUP resorder 192.168.186.105:8000 register finished
2023-11-24 18:37:46.339  INFO 27024 --- [main] com.yc.ResorderApp                       : Started ResorderApp in 3.165 seconds (JVM running for 3.724)
2023-11-24 18:43:13.441  INFO 27024 --- [http-nio-8000-exec-2] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
2023-11-24 18:43:13.441  INFO 27024 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2023-11-24 18:43:13.442 DEBUG 27024 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : Detected StandardServletMultipartResolver
2023-11-24 18:43:13.442 DEBUG 27024 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : Detected AcceptHeaderLocaleResolver
2023-11-24 18:43:13.442 DEBUG 27024 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : Detected FixedThemeResolver
2023-11-24 18:43:13.442 DEBUG 27024 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : Detected org.springframework.web.servlet.view.DefaultRequestToViewNameTranslator@50af55b4
2023-11-24 18:43:13.443 DEBUG 27024 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : Detected org.springframework.web.servlet.support.SessionFlashMapManager@34c17727
2023-11-24 18:43:13.443 DEBUG 27024 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : enableLoggingRequestDetails='false': request parameters and headers will be masked to prevent unsafe logging of potentially sensitive data
2023-11-24 18:43:13.443  INFO 27024 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : Completed initialization in 2 ms
2023-11-24 18:43:13.450 DEBUG 27024 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : GET "/resorder/addCart?fid=2&num=1", parameters={masked}
2023-11-24 18:43:13.455 DEBUG 27024 --- [http-nio-8000-exec-2] s.w.s.m.m.a.RequestMappingHandlerMapping : Mapped to com.yc.web.controller.ResorderController#addCart(Integer, Integer, HttpSession)
2023-11-24 18:43:13.488 DEBUG 27024 --- [http-nio-8000-exec-2] o.s.web.client.RestTemplate              : HTTP GET http://localhost:9000/resfood/findById/2
2023-11-24 18:43:13.491 DEBUG 27024 --- [http-nio-8000-exec-2] o.s.web.client.RestTemplate              : Accept=[application/json, application/*+json]
2023-11-24 18:43:13.736 DEBUG 27024 --- [http-nio-8000-exec-2] o.s.web.client.RestTemplate              : Response 200 OK
2023-11-24 18:43:13.737 DEBUG 27024 --- [http-nio-8000-exec-2] o.s.web.client.RestTemplate              : Reading to [java.util.Map<?, ?>]
2023-11-24 18:43:13.744  INFO 27024 --- [http-nio-8000-exec-2] c.yc.web.controller.ResorderController   : 发送请求后得到商品信息：{fid=2, fname=蛋炒饭, normprice=22.0, realprice=20.0, detail=营养丰富, fphoto=500022.jpg, detail_count=null}
2023-11-24 18:43:13.750 DEBUG 27024 --- [http-nio-8000-exec-2] m.m.a.RequestResponseBodyMethodProcessor : Using 'application/json', given [*/*] and supported [application/json, application/*+json, application/json, application/*+json]
2023-11-24 18:43:13.750 DEBUG 27024 --- [http-nio-8000-exec-2] m.m.a.RequestResponseBodyMethodProcessor : Writing [{}]
2023-11-24 18:43:13.757 DEBUG 27024 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : Completed 200 OK
2023-11-24 19:16:00.451  INFO 5428 --- [main] c.a.n.client.env.SearchableProperties    : properties search order:PROPERTIES->JVM->ENV->DEFAULT_SETTING
2023-11-24 19:16:00.541  INFO 5428 --- [main] com.yc.ResorderApp                       : Starting ResorderApp using Java 1.8.0_372 on DESKTOP-2GHSLBQ with PID 5428 (D:\workspaceYC134\res134_cloud\res_order\target\classes started by xinye in D:\workspaceYC134\res134_cloud)
2023-11-24 19:16:00.542  INFO 5428 --- [main] com.yc.ResorderApp                       : No active profile set, falling back to 1 default profile: "default"
2023-11-24 19:16:00.991  INFO 5428 --- [main] o.s.cloud.context.scope.GenericScope     : BeanFactory id=4be545fb-e62a-38cb-95e3-6da9256719a2
2023-11-24 19:16:01.150  INFO 5428 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8000 (http)
2023-11-24 19:16:01.156  INFO 5428 --- [main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2023-11-24 19:16:01.156  INFO 5428 --- [main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.63]
2023-11-24 19:16:01.234  INFO 5428 --- [main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2023-11-24 19:16:01.234  INFO 5428 --- [main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 668 ms
2023-11-24 19:16:01.361 DEBUG 5428 --- [main] s.w.s.m.m.a.RequestMappingHandlerAdapter : ControllerAdvice beans: 0 @ModelAttribute, 0 @InitBinder, 1 RequestBodyAdvice, 1 ResponseBodyAdvice
2023-11-24 19:16:01.399 DEBUG 5428 --- [main] s.w.s.m.m.a.RequestMappingHandlerMapping : 3 mappings in 'requestMappingHandlerMapping'
2023-11-24 19:16:01.408 DEBUG 5428 --- [main] o.s.w.s.handler.SimpleUrlHandlerMapping  : Patterns [/webjars/**, /**] in 'resourceHandlerMapping'
2023-11-24 19:16:01.413 DEBUG 5428 --- [main] .m.m.a.ExceptionHandlerExceptionResolver : ControllerAdvice beans: 0 @ExceptionHandler, 1 ResponseBodyAdvice
2023-11-24 19:16:02.027  WARN 5428 --- [main] iguration$LoadBalancerCaffeineWarnLogger : Spring Cloud LoadBalancer is currently working with the default cache. While this cache implementation is useful for development and tests, it's recommended to use Caffeine cache in production.You can switch to using Caffeine cache, by adding it and org.springframework.cache.caffeine.CaffeineCacheManager to the classpath.
2023-11-24 19:16:02.072  INFO 5428 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 8000 (http) with context path ''
2023-11-24 19:16:02.106  INFO 5428 --- [main] c.a.n.p.a.s.c.ClientAuthPluginManager    : [ClientAuthPluginManager] Load ClientAuthService com.alibaba.nacos.client.auth.impl.NacosClientAuthServiceImpl success.
2023-11-24 19:16:02.106  INFO 5428 --- [main] c.a.n.p.a.s.c.ClientAuthPluginManager    : [ClientAuthPluginManager] Load ClientAuthService com.alibaba.nacos.client.auth.ram.RamClientAuthServiceImpl success.
2023-11-24 19:16:03.181  INFO 5428 --- [main] c.a.c.n.registry.NacosServiceRegistry    : nacos registry, DEFAULT_GROUP resorder 192.168.186.105:8000 register finished
2023-11-24 19:16:03.386  INFO 5428 --- [main] com.yc.ResorderApp                       : Started ResorderApp in 3.432 seconds (JVM running for 4.027)
2023-11-24 19:19:35.830  INFO 5428 --- [http-nio-8000-exec-2] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
2023-11-24 19:19:35.831  INFO 5428 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2023-11-24 19:19:35.831 DEBUG 5428 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : Detected StandardServletMultipartResolver
2023-11-24 19:19:35.831 DEBUG 5428 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : Detected AcceptHeaderLocaleResolver
2023-11-24 19:19:35.831 DEBUG 5428 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : Detected FixedThemeResolver
2023-11-24 19:19:35.832 DEBUG 5428 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : Detected org.springframework.web.servlet.view.DefaultRequestToViewNameTranslator@bf2f4de
2023-11-24 19:19:35.832 DEBUG 5428 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : Detected org.springframework.web.servlet.support.SessionFlashMapManager@5a3c531d
2023-11-24 19:19:35.832 DEBUG 5428 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : enableLoggingRequestDetails='false': request parameters and headers will be masked to prevent unsafe logging of potentially sensitive data
2023-11-24 19:19:35.832  INFO 5428 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : Completed initialization in 1 ms
2023-11-24 19:19:35.838 DEBUG 5428 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : GET "/resorder/addCart?fid=2&num=1", parameters={masked}
2023-11-24 19:19:35.842 DEBUG 5428 --- [http-nio-8000-exec-2] s.w.s.m.m.a.RequestMappingHandlerMapping : Mapped to com.yc.web.controller.ResorderController#addCart(Integer, Integer, HttpSession)
2023-11-24 19:19:35.868 DEBUG 5428 --- [http-nio-8000-exec-2] o.s.web.client.RestTemplate              : HTTP GET http://resfood/resfood/findById/2
2023-11-24 19:19:35.872 DEBUG 5428 --- [http-nio-8000-exec-2] o.s.web.client.RestTemplate              : Accept=[application/json, application/*+json]
2023-11-24 19:19:36.329 DEBUG 5428 --- [http-nio-8000-exec-2] o.s.web.client.RestTemplate              : Response 200 OK
2023-11-24 19:19:36.330 DEBUG 5428 --- [http-nio-8000-exec-2] o.s.web.client.RestTemplate              : Reading to [java.util.Map<?, ?>]
2023-11-24 19:19:36.335  INFO 5428 --- [http-nio-8000-exec-2] c.yc.web.controller.ResorderController   : 发送请求后得到商品信息：{fid=2, fname=蛋炒饭, normprice=22.0, realprice=20.0, detail=营养丰富, fphoto=500022.jpg, detail_count=null}
2023-11-24 19:19:36.340 DEBUG 5428 --- [http-nio-8000-exec-2] m.m.a.RequestResponseBodyMethodProcessor : Using 'application/json', given [*/*] and supported [application/json, application/*+json, application/json, application/*+json]
2023-11-24 19:19:36.340 DEBUG 5428 --- [http-nio-8000-exec-2] m.m.a.RequestResponseBodyMethodProcessor : Writing [{}]
2023-11-24 19:19:36.347 DEBUG 5428 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : Completed 200 OK
2023-11-24 19:19:43.846 DEBUG 5428 --- [http-nio-8000-exec-7] o.s.web.servlet.DispatcherServlet        : GET "/resorder/addCart?fid=2&num=1", parameters={masked}
2023-11-24 19:19:43.846 DEBUG 5428 --- [http-nio-8000-exec-7] s.w.s.m.m.a.RequestMappingHandlerMapping : Mapped to com.yc.web.controller.ResorderController#addCart(Integer, Integer, HttpSession)
2023-11-24 19:19:43.848 DEBUG 5428 --- [http-nio-8000-exec-7] o.s.web.client.RestTemplate              : HTTP GET http://resfood/resfood/findById/2
2023-11-24 19:19:43.848 DEBUG 5428 --- [http-nio-8000-exec-7] o.s.web.client.RestTemplate              : Accept=[application/json, application/*+json]
2023-11-24 19:19:43.863 DEBUG 5428 --- [http-nio-8000-exec-7] o.s.web.client.RestTemplate              : Response 200 OK
2023-11-24 19:19:43.864 DEBUG 5428 --- [http-nio-8000-exec-7] o.s.web.client.RestTemplate              : Reading to [java.util.Map<?, ?>]
2023-11-24 19:19:43.864  INFO 5428 --- [http-nio-8000-exec-7] c.yc.web.controller.ResorderController   : 发送请求后得到商品信息：{fid=2, fname=蛋炒饭, normprice=22.0, realprice=20.0, detail=营养丰富, fphoto=500022.jpg, detail_count=null}
2023-11-24 19:19:43.864 DEBUG 5428 --- [http-nio-8000-exec-7] m.m.a.RequestResponseBodyMethodProcessor : Using 'application/json', given [*/*] and supported [application/json, application/*+json, application/json, application/*+json]
2023-11-24 19:19:43.865 DEBUG 5428 --- [http-nio-8000-exec-7] m.m.a.RequestResponseBodyMethodProcessor : Writing [{}]
2023-11-24 19:19:43.865 DEBUG 5428 --- [http-nio-8000-exec-7] o.s.web.servlet.DispatcherServlet        : Completed 200 OK
2023-11-24 19:19:51.439 DEBUG 5428 --- [http-nio-8000-exec-4] o.s.web.servlet.DispatcherServlet        : GET "/resorder/addCart?fid=2&num=1", parameters={masked}
2023-11-24 19:19:51.439 DEBUG 5428 --- [http-nio-8000-exec-4] s.w.s.m.m.a.RequestMappingHandlerMapping : Mapped to com.yc.web.controller.ResorderController#addCart(Integer, Integer, HttpSession)
2023-11-24 19:19:51.440 DEBUG 5428 --- [http-nio-8000-exec-4] o.s.web.client.RestTemplate              : HTTP GET http://resfood/resfood/findById/2
2023-11-24 19:19:51.440 DEBUG 5428 --- [http-nio-8000-exec-4] o.s.web.client.RestTemplate              : Accept=[application/json, application/*+json]
2023-11-24 19:19:51.454 DEBUG 5428 --- [http-nio-8000-exec-4] o.s.web.client.RestTemplate              : Response 200 OK
2023-11-24 19:19:51.454 DEBUG 5428 --- [http-nio-8000-exec-4] o.s.web.client.RestTemplate              : Reading to [java.util.Map<?, ?>]
2023-11-24 19:19:51.454  INFO 5428 --- [http-nio-8000-exec-4] c.yc.web.controller.ResorderController   : 发送请求后得到商品信息：{fid=2, fname=蛋炒饭, normprice=22.0, realprice=20.0, detail=营养丰富, fphoto=500022.jpg, detail_count=null}
2023-11-24 19:19:51.455 DEBUG 5428 --- [http-nio-8000-exec-4] m.m.a.RequestResponseBodyMethodProcessor : Using 'application/json', given [*/*] and supported [application/json, application/*+json, application/json, application/*+json]
2023-11-24 19:19:51.455 DEBUG 5428 --- [http-nio-8000-exec-4] m.m.a.RequestResponseBodyMethodProcessor : Writing [{}]
2023-11-24 19:19:51.455 DEBUG 5428 --- [http-nio-8000-exec-4] o.s.web.servlet.DispatcherServlet        : Completed 200 OK
2023-11-24 20:40:28.125  INFO 4536 --- [main] c.a.n.client.env.SearchableProperties    : properties search order:PROPERTIES->JVM->ENV->DEFAULT_SETTING
2023-11-24 20:40:28.218  INFO 4536 --- [main] com.yc.ResorderApp                       : Starting ResorderApp using Java 1.8.0_372 on DESKTOP-2GHSLBQ with PID 4536 (D:\workspaceYC134\res134_cloud\res_order\target\classes started by xinye in D:\workspaceYC134\res134_cloud)
2023-11-24 20:40:28.219  INFO 4536 --- [main] com.yc.ResorderApp                       : No active profile set, falling back to 1 default profile: "default"
2023-11-24 20:40:28.757  INFO 4536 --- [main] o.s.cloud.context.scope.GenericScope     : BeanFactory id=42bdc39e-2c37-3730-97d6-358fd4b1947c
2023-11-24 20:40:28.910  INFO 4536 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8000 (http)
2023-11-24 20:40:28.916  INFO 4536 --- [main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2023-11-24 20:40:28.916  INFO 4536 --- [main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.63]
2023-11-24 20:40:28.991  INFO 4536 --- [main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2023-11-24 20:40:28.991  INFO 4536 --- [main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 742 ms
2023-11-24 20:40:29.140 DEBUG 4536 --- [main] s.w.s.m.m.a.RequestMappingHandlerAdapter : ControllerAdvice beans: 0 @ModelAttribute, 0 @InitBinder, 1 RequestBodyAdvice, 1 ResponseBodyAdvice
2023-11-24 20:40:29.179 DEBUG 4536 --- [main] s.w.s.m.m.a.RequestMappingHandlerMapping : 3 mappings in 'requestMappingHandlerMapping'
2023-11-24 20:40:29.189 DEBUG 4536 --- [main] o.s.w.s.handler.SimpleUrlHandlerMapping  : Patterns [/webjars/**, /**] in 'resourceHandlerMapping'
2023-11-24 20:40:29.194 DEBUG 4536 --- [main] .m.m.a.ExceptionHandlerExceptionResolver : ControllerAdvice beans: 0 @ExceptionHandler, 1 ResponseBodyAdvice
2023-11-24 20:40:29.810  WARN 4536 --- [main] iguration$LoadBalancerCaffeineWarnLogger : Spring Cloud LoadBalancer is currently working with the default cache. While this cache implementation is useful for development and tests, it's recommended to use Caffeine cache in production.You can switch to using Caffeine cache, by adding it and org.springframework.cache.caffeine.CaffeineCacheManager to the classpath.
2023-11-24 20:40:29.857  INFO 4536 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 8000 (http) with context path ''
2023-11-24 20:40:29.885  INFO 4536 --- [main] c.a.n.p.a.s.c.ClientAuthPluginManager    : [ClientAuthPluginManager] Load ClientAuthService com.alibaba.nacos.client.auth.impl.NacosClientAuthServiceImpl success.
2023-11-24 20:40:29.885  INFO 4536 --- [main] c.a.n.p.a.s.c.ClientAuthPluginManager    : [ClientAuthPluginManager] Load ClientAuthService com.alibaba.nacos.client.auth.ram.RamClientAuthServiceImpl success.
2023-11-24 20:40:30.903  INFO 4536 --- [main] c.a.c.n.registry.NacosServiceRegistry    : nacos registry, DEFAULT_GROUP resorder 192.168.186.105:8000 register finished
2023-11-24 20:40:31.117  INFO 4536 --- [main] com.yc.ResorderApp                       : Started ResorderApp in 3.481 seconds (JVM running for 4.195)
2023-11-24 20:40:35.359  INFO 4536 --- [http-nio-8000-exec-2] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
2023-11-24 20:40:35.360  INFO 4536 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2023-11-24 20:40:35.360 DEBUG 4536 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : Detected StandardServletMultipartResolver
2023-11-24 20:40:35.360 DEBUG 4536 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : Detected AcceptHeaderLocaleResolver
2023-11-24 20:40:35.360 DEBUG 4536 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : Detected FixedThemeResolver
2023-11-24 20:40:35.361 DEBUG 4536 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : Detected org.springframework.web.servlet.view.DefaultRequestToViewNameTranslator@4f09abe2
2023-11-24 20:40:35.361 DEBUG 4536 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : Detected org.springframework.web.servlet.support.SessionFlashMapManager@1dd6e9cb
2023-11-24 20:40:35.361 DEBUG 4536 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : enableLoggingRequestDetails='false': request parameters and headers will be masked to prevent unsafe logging of potentially sensitive data
2023-11-24 20:40:35.361  INFO 4536 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : Completed initialization in 1 ms
2023-11-24 20:40:35.371 DEBUG 4536 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : GET "/resorder/addCart?fid=2&num=1", parameters={masked}
2023-11-24 20:40:35.378 DEBUG 4536 --- [http-nio-8000-exec-2] s.w.s.m.m.a.RequestMappingHandlerMapping : Mapped to com.yc.web.controller.ResorderController#addCart(Integer, Integer, HttpSession)
2023-11-24 20:40:35.430 DEBUG 4536 --- [http-nio-8000-exec-2] o.s.web.client.RestTemplate              : HTTP GET http://resfood/resfood/findById/2
2023-11-24 20:40:35.434 DEBUG 4536 --- [http-nio-8000-exec-2] o.s.web.client.RestTemplate              : Accept=[application/json, application/*+json]
2023-11-24 20:40:35.485 DEBUG 4536 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : Failed to complete request: java.lang.NullPointerException
2023-11-24 20:40:35.490 ERROR 4536 --- [http-nio-8000-exec-2] o.a.c.c.C.[.[.[/].[dispatcherServlet]    : Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed; nested exception is java.lang.NullPointerException] with root cause

java.lang.NullPointerException: null
	at java.util.concurrent.ConcurrentHashMap.get(ConcurrentHashMap.java:936) ~[na:1.8.0_372]
	at java.util.concurrent.ConcurrentHashMap.containsKey(ConcurrentHashMap.java:964) ~[na:1.8.0_372]
	at org.springframework.cloud.context.named.NamedContextFactory.getContext(NamedContextFactory.java:105) ~[spring-cloud-context-3.1.6.jar:3.1.6]
	at org.springframework.cloud.context.named.NamedContextFactory.getProvider(NamedContextFactory.java:179) ~[spring-cloud-context-3.1.6.jar:3.1.6]
	at org.springframework.cloud.context.named.ClientFactoryObjectProvider.delegate(ClientFactoryObjectProvider.java:115) ~[spring-cloud-context-3.1.6.jar:3.1.6]
	at org.springframework.cloud.context.named.ClientFactoryObjectProvider.getIfAvailable(ClientFactoryObjectProvider.java:64) ~[spring-cloud-context-3.1.6.jar:3.1.6]
	at com.yc.config.OnlyOneLoadBalancer.choose(OnlyOneLoadBalancer.java:36) ~[classes/:na]
	at com.yc.config.OnlyOneLoadBalancer.choose(OnlyOneLoadBalancer.java:24) ~[classes/:na]
	at org.springframework.cloud.loadbalancer.blocking.client.BlockingLoadBalancerClient.choose(BlockingLoadBalancerClient.java:178) ~[spring-cloud-loadbalancer-3.1.6.jar:3.1.6]
	at org.springframework.cloud.loadbalancer.blocking.client.BlockingLoadBalancerClient.execute(BlockingLoadBalancerClient.java:85) ~[spring-cloud-loadbalancer-3.1.6.jar:3.1.6]
	at org.springframework.cloud.client.loadbalancer.LoadBalancerInterceptor.intercept(LoadBalancerInterceptor.java:56) ~[spring-cloud-commons-3.1.6.jar:3.1.6]
	at org.springframework.http.client.InterceptingClientHttpRequest$InterceptingRequestExecution.execute(InterceptingClientHttpRequest.java:93) ~[spring-web-5.3.20.jar:5.3.20]
	at org.springframework.http.client.InterceptingClientHttpRequest.executeInternal(InterceptingClientHttpRequest.java:77) ~[spring-web-5.3.20.jar:5.3.20]
	at org.springframework.http.client.AbstractBufferingClientHttpRequest.executeInternal(AbstractBufferingClientHttpRequest.java:48) ~[spring-web-5.3.20.jar:5.3.20]
	at org.springframework.http.client.AbstractClientHttpRequest.execute(AbstractClientHttpRequest.java:66) ~[spring-web-5.3.20.jar:5.3.20]
	at org.springframework.web.client.RestTemplate.doExecute(RestTemplate.java:776) ~[spring-web-5.3.20.jar:5.3.20]
	at org.springframework.web.client.RestTemplate.execute(RestTemplate.java:711) ~[spring-web-5.3.20.jar:5.3.20]
	at org.springframework.web.client.RestTemplate.getForObject(RestTemplate.java:334) ~[spring-web-5.3.20.jar:5.3.20]
	at com.yc.web.controller.ResorderController.addCart(ResorderController.java:30) ~[classes/:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_372]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_372]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_372]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_372]
	at org.springframework.web.method.support.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:205) ~[spring-web-5.3.20.jar:5.3.20]
	at org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest(InvocableHandlerMethod.java:150) ~[spring-web-5.3.20.jar:5.3.20]
	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:117) ~[spring-webmvc-5.3.20.jar:5.3.20]
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:895) ~[spring-webmvc-5.3.20.jar:5.3.20]
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:808) ~[spring-webmvc-5.3.20.jar:5.3.20]
	at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:87) ~[spring-webmvc-5.3.20.jar:5.3.20]
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1067) ~[spring-webmvc-5.3.20.jar:5.3.20]
	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:963) ~[spring-webmvc-5.3.20.jar:5.3.20]
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1006) ~[spring-webmvc-5.3.20.jar:5.3.20]
	at org.springframework.web.servlet.FrameworkServlet.doGet(FrameworkServlet.java:898) ~[spring-webmvc-5.3.20.jar:5.3.20]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:655) ~[tomcat-embed-core-9.0.63.jar:4.0.FR]
	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:883) ~[spring-webmvc-5.3.20.jar:5.3.20]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764) ~[tomcat-embed-core-9.0.63.jar:4.0.FR]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227) ~[tomcat-embed-core-9.0.63.jar:9.0.63]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) ~[tomcat-embed-core-9.0.63.jar:9.0.63]
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53) ~[tomcat-embed-websocket-9.0.63.jar:9.0.63]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189) ~[tomcat-embed-core-9.0.63.jar:9.0.63]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) ~[tomcat-embed-core-9.0.63.jar:9.0.63]
	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:100) ~[spring-web-5.3.20.jar:5.3.20]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117) ~[spring-web-5.3.20.jar:5.3.20]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189) ~[tomcat-embed-core-9.0.63.jar:9.0.63]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) ~[tomcat-embed-core-9.0.63.jar:9.0.63]
	at org.springframework.web.filter.FormContentFilter.doFilterInternal(FormContentFilter.java:93) ~[spring-web-5.3.20.jar:5.3.20]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117) ~[spring-web-5.3.20.jar:5.3.20]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189) ~[tomcat-embed-core-9.0.63.jar:9.0.63]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) ~[tomcat-embed-core-9.0.63.jar:9.0.63]
	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:201) ~[spring-web-5.3.20.jar:5.3.20]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117) ~[spring-web-5.3.20.jar:5.3.20]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189) ~[tomcat-embed-core-9.0.63.jar:9.0.63]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) ~[tomcat-embed-core-9.0.63.jar:9.0.63]
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197) ~[tomcat-embed-core-9.0.63.jar:9.0.63]
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97) [tomcat-embed-core-9.0.63.jar:9.0.63]
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541) [tomcat-embed-core-9.0.63.jar:9.0.63]
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135) [tomcat-embed-core-9.0.63.jar:9.0.63]
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92) [tomcat-embed-core-9.0.63.jar:9.0.63]
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78) [tomcat-embed-core-9.0.63.jar:9.0.63]
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:360) [tomcat-embed-core-9.0.63.jar:9.0.63]
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399) [tomcat-embed-core-9.0.63.jar:9.0.63]
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65) [tomcat-embed-core-9.0.63.jar:9.0.63]
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:890) [tomcat-embed-core-9.0.63.jar:9.0.63]
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1743) [tomcat-embed-core-9.0.63.jar:9.0.63]
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) [tomcat-embed-core-9.0.63.jar:9.0.63]
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191) [tomcat-embed-core-9.0.63.jar:9.0.63]
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659) [tomcat-embed-core-9.0.63.jar:9.0.63]
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) [tomcat-embed-core-9.0.63.jar:9.0.63]
	at java.lang.Thread.run(Thread.java:750) [na:1.8.0_372]

2023-11-24 20:40:35.494 DEBUG 4536 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : "ERROR" dispatch for GET "/error?fid=2&num=1", parameters={masked}
2023-11-24 20:40:35.495 DEBUG 4536 --- [http-nio-8000-exec-2] s.w.s.m.m.a.RequestMappingHandlerMapping : Mapped to org.springframework.boot.autoconfigure.web.servlet.error.BasicErrorController#error(HttpServletRequest)
2023-11-24 20:40:35.500 DEBUG 4536 --- [http-nio-8000-exec-2] o.s.w.s.m.m.a.HttpEntityMethodProcessor  : Using 'application/json', given [*/*] and supported [application/json, application/*+json, application/json, application/*+json]
2023-11-24 20:40:35.501 DEBUG 4536 --- [http-nio-8000-exec-2] o.s.w.s.m.m.a.HttpEntityMethodProcessor  : Writing [{timestamp=Fri Nov 24 20:40:35 CST 2023, status=500, error=Internal Server Error, path=/resorder/add (truncated)...]
2023-11-24 20:40:35.509 DEBUG 4536 --- [http-nio-8000-exec-2] o.s.web.servlet.DispatcherServlet        : Exiting from "ERROR" dispatch, status 500
2023-11-24 20:43:14.102  INFO 740 --- [main] c.a.n.client.env.SearchableProperties    : properties search order:PROPERTIES->JVM->ENV->DEFAULT_SETTING
2023-11-24 20:43:14.187  INFO 740 --- [main] com.yc.ResorderApp                       : Starting ResorderApp using Java 1.8.0_372 on DESKTOP-2GHSLBQ with PID 740 (D:\workspaceYC134\res134_cloud\res_order\target\classes started by xinye in D:\workspaceYC134\res134_cloud)
2023-11-24 20:43:14.188  INFO 740 --- [main] com.yc.ResorderApp                       : No active profile set, falling back to 1 default profile: "default"
2023-11-24 20:43:14.641  INFO 740 --- [main] o.s.cloud.context.scope.GenericScope     : BeanFactory id=42bdc39e-2c37-3730-97d6-358fd4b1947c
2023-11-24 20:43:14.793  INFO 740 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8000 (http)
2023-11-24 20:43:14.798  INFO 740 --- [main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2023-11-24 20:43:14.798  INFO 740 --- [main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.63]
2023-11-24 20:43:14.871  INFO 740 --- [main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2023-11-24 20:43:14.872  INFO 740 --- [main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 659 ms
2023-11-24 20:43:15.012 DEBUG 740 --- [main] s.w.s.m.m.a.RequestMappingHandlerAdapter : ControllerAdvice beans: 0 @ModelAttribute, 0 @InitBinder, 1 RequestBodyAdvice, 1 ResponseBodyAdvice
2023-11-24 20:43:15.050 DEBUG 740 --- [main] s.w.s.m.m.a.RequestMappingHandlerMapping : 3 mappings in 'requestMappingHandlerMapping'
2023-11-24 20:43:15.060 DEBUG 740 --- [main] o.s.w.s.handler.SimpleUrlHandlerMapping  : Patterns [/webjars/**, /**] in 'resourceHandlerMapping'
2023-11-24 20:43:15.065 DEBUG 740 --- [main] .m.m.a.ExceptionHandlerExceptionResolver : ControllerAdvice beans: 0 @ExceptionHandler, 1 ResponseBodyAdvice
2023-11-24 20:43:15.680  WARN 740 --- [main] iguration$LoadBalancerCaffeineWarnLogger : Spring Cloud LoadBalancer is currently working with the default cache. While this cache implementation is useful for development and tests, it's recommended to use Caffeine cache in production.You can switch to using Caffeine cache, by adding it and org.springframework.cache.caffeine.CaffeineCacheManager to the classpath.
2023-11-24 20:43:15.724  INFO 740 --- [main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 8000 (http) with context path ''
2023-11-24 20:43:15.752  INFO 740 --- [main] c.a.n.p.a.s.c.ClientAuthPluginManager    : [ClientAuthPluginManager] Load ClientAuthService com.alibaba.nacos.client.auth.impl.NacosClientAuthServiceImpl success.
2023-11-24 20:43:15.752  INFO 740 --- [main] c.a.n.p.a.s.c.ClientAuthPluginManager    : [ClientAuthPluginManager] Load ClientAuthService com.alibaba.nacos.client.auth.ram.RamClientAuthServiceImpl success.
2023-11-24 20:43:16.803  INFO 740 --- [main] c.a.c.n.registry.NacosServiceRegistry    : nacos registry, DEFAULT_GROUP resorder 192.168.186.105:8000 register finished
2023-11-24 20:43:17.016  INFO 740 --- [main] com.yc.ResorderApp                       : Started ResorderApp in 3.394 seconds (JVM running for 4.076)
2023-11-24 20:43:19.689  INFO 740 --- [http-nio-8000-exec-8] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
2023-11-24 20:43:19.689  INFO 740 --- [http-nio-8000-exec-8] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2023-11-24 20:43:19.690 DEBUG 740 --- [http-nio-8000-exec-8] o.s.web.servlet.DispatcherServlet        : Detected StandardServletMultipartResolver
2023-11-24 20:43:19.690 DEBUG 740 --- [http-nio-8000-exec-8] o.s.web.servlet.DispatcherServlet        : Detected AcceptHeaderLocaleResolver
2023-11-24 20:43:19.690 DEBUG 740 --- [http-nio-8000-exec-8] o.s.web.servlet.DispatcherServlet        : Detected FixedThemeResolver
2023-11-24 20:43:19.690 DEBUG 740 --- [http-nio-8000-exec-8] o.s.web.servlet.DispatcherServlet        : Detected org.springframework.web.servlet.view.DefaultRequestToViewNameTranslator@693c1e
2023-11-24 20:43:19.691 DEBUG 740 --- [http-nio-8000-exec-8] o.s.web.servlet.DispatcherServlet        : Detected org.springframework.web.servlet.support.SessionFlashMapManager@34b9f0dd
2023-11-24 20:43:19.691 DEBUG 740 --- [http-nio-8000-exec-8] o.s.web.servlet.DispatcherServlet        : enableLoggingRequestDetails='false': request parameters and headers will be masked to prevent unsafe logging of potentially sensitive data
2023-11-24 20:43:19.691  INFO 740 --- [http-nio-8000-exec-8] o.s.web.servlet.DispatcherServlet        : Completed initialization in 2 ms
2023-11-24 20:43:19.697 DEBUG 740 --- [http-nio-8000-exec-8] o.s.web.servlet.DispatcherServlet        : GET "/resorder/addCart?fid=2&num=1", parameters={masked}
2023-11-24 20:43:19.702 DEBUG 740 --- [http-nio-8000-exec-8] s.w.s.m.m.a.RequestMappingHandlerMapping : Mapped to com.yc.web.controller.ResorderController#addCart(Integer, Integer, HttpSession)
2023-11-24 20:43:19.739 DEBUG 740 --- [http-nio-8000-exec-8] o.s.web.client.RestTemplate              : HTTP GET http://resfood/resfood/findById/2
2023-11-24 20:43:19.742 DEBUG 740 --- [http-nio-8000-exec-8] o.s.web.client.RestTemplate              : Accept=[application/json, application/*+json]
2023-11-24 20:43:19.841  INFO 740 --- [http-nio-8000-exec-8] com.yc.config.OnlyOneLoadBalancer        : 自定义负载
2023-11-24 20:43:19.878 DEBUG 740 --- [http-nio-8000-exec-8] o.s.web.client.RestTemplate              : Response 200 OK
2023-11-24 20:43:19.879 DEBUG 740 --- [http-nio-8000-exec-8] o.s.web.client.RestTemplate              : Reading to [java.util.Map<?, ?>]
2023-11-24 20:43:19.884  INFO 740 --- [http-nio-8000-exec-8] c.yc.web.controller.ResorderController   : 发送请求后得到商品信息：{fid=2, fname=蛋炒饭, normprice=22.0, realprice=20.0, detail=营养丰富, fphoto=500022.jpg, detail_count=null}
2023-11-24 20:43:19.889 DEBUG 740 --- [http-nio-8000-exec-8] m.m.a.RequestResponseBodyMethodProcessor : Using 'application/json', given [*/*] and supported [application/json, application/*+json, application/json, application/*+json]
2023-11-24 20:43:19.890 DEBUG 740 --- [http-nio-8000-exec-8] m.m.a.RequestResponseBodyMethodProcessor : Writing [{}]
2023-11-24 20:43:19.896 DEBUG 740 --- [http-nio-8000-exec-8] o.s.web.servlet.DispatcherServlet        : Completed 200 OK
2023-11-24 20:49:13.711 DEBUG 740 --- [http-nio-8000-exec-7] o.s.web.servlet.DispatcherServlet        : GET "/resorder/addCart?fid=2&num=1", parameters={masked}
2023-11-24 20:49:13.711 DEBUG 740 --- [http-nio-8000-exec-7] s.w.s.m.m.a.RequestMappingHandlerMapping : Mapped to com.yc.web.controller.ResorderController#addCart(Integer, Integer, HttpSession)
2023-11-24 20:49:13.715 DEBUG 740 --- [http-nio-8000-exec-7] o.s.web.client.RestTemplate              : HTTP GET http://resfood/resfood/findById/2
2023-11-24 20:49:13.715 DEBUG 740 --- [http-nio-8000-exec-7] o.s.web.client.RestTemplate              : Accept=[application/json, application/*+json]
2023-11-24 20:49:13.716  INFO 740 --- [http-nio-8000-exec-7] com.yc.config.OnlyOneLoadBalancer        : 自定义负载
2023-11-24 20:49:13.729 DEBUG 740 --- [http-nio-8000-exec-7] o.s.web.client.RestTemplate              : Response 200 OK
2023-11-24 20:49:13.729 DEBUG 740 --- [http-nio-8000-exec-7] o.s.web.client.RestTemplate              : Reading to [java.util.Map<?, ?>]
2023-11-24 20:49:13.730  INFO 740 --- [http-nio-8000-exec-7] c.yc.web.controller.ResorderController   : 发送请求后得到商品信息：{fid=2, fname=蛋炒饭, normprice=22.0, realprice=20.0, detail=营养丰富, fphoto=500022.jpg, detail_count=null}
2023-11-24 20:49:13.730 DEBUG 740 --- [http-nio-8000-exec-7] m.m.a.RequestResponseBodyMethodProcessor : Using 'application/json', given [*/*] and supported [application/json, application/*+json, application/json, application/*+json]
2023-11-24 20:49:13.730 DEBUG 740 --- [http-nio-8000-exec-7] m.m.a.RequestResponseBodyMethodProcessor : Writing [{}]
2023-11-24 20:49:13.731 DEBUG 740 --- [http-nio-8000-exec-7] o.s.web.servlet.DispatcherServlet        : Completed 200 OK
2023-11-24 21:30:08.904  INFO 8292 --- [lettuce-nioEventLoop-4-2] io.lettuce.core.protocol.CommandHandler  : null Unexpected exception during request: java.io.IOException: 远程主机强迫关闭了一个现有的连接。

java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method) ~[na:1.8.0_372]
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43) ~[na:1.8.0_372]
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223) ~[na:1.8.0_372]
	at sun.nio.ch.IOUtil.read(IOUtil.java:192) ~[na:1.8.0_372]
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:379) ~[na:1.8.0_372]
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:258) ~[netty-buffer-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132) ~[netty-buffer-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357) ~[netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151) ~[netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:722) [netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:658) [netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:584) [netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:496) [netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:995) [netty-common-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-common-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-common-4.1.77.Final.jar:4.1.77.Final]
	at java.lang.Thread.run(Thread.java:750) [na:1.8.0_372]

2023-11-24 21:30:08.904  INFO 8292 --- [lettuce-nioEventLoop-4-1] io.lettuce.core.protocol.CommandHandler  : null Unexpected exception during request: java.io.IOException: 远程主机强迫关闭了一个现有的连接。

java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method) ~[na:1.8.0_372]
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43) ~[na:1.8.0_372]
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223) ~[na:1.8.0_372]
	at sun.nio.ch.IOUtil.read(IOUtil.java:192) ~[na:1.8.0_372]
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:379) ~[na:1.8.0_372]
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:258) ~[netty-buffer-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132) ~[netty-buffer-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357) ~[netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151) ~[netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:722) [netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:658) [netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:584) [netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:496) [netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:995) [netty-common-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-common-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-common-4.1.77.Final.jar:4.1.77.Final]
	at java.lang.Thread.run(Thread.java:750) [na:1.8.0_372]

2023-11-24 21:30:08.904  INFO 8292 --- [lettuce-nioEventLoop-4-3] io.lettuce.core.protocol.CommandHandler  : null Unexpected exception during request: java.io.IOException: 远程主机强迫关闭了一个现有的连接。

java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method) ~[na:1.8.0_372]
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43) ~[na:1.8.0_372]
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223) ~[na:1.8.0_372]
	at sun.nio.ch.IOUtil.read(IOUtil.java:192) ~[na:1.8.0_372]
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:379) ~[na:1.8.0_372]
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:258) ~[netty-buffer-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132) ~[netty-buffer-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357) ~[netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151) ~[netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:722) [netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:658) [netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:584) [netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:496) [netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:995) [netty-common-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-common-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-common-4.1.77.Final.jar:4.1.77.Final]
	at java.lang.Thread.run(Thread.java:750) [na:1.8.0_372]

2023-11-24 21:30:08.905  INFO 1188 --- [lettuce-nioEventLoop-4-3] io.lettuce.core.protocol.CommandHandler  : null Unexpected exception during request: java.io.IOException: 远程主机强迫关闭了一个现有的连接。

java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method) ~[na:1.8.0_372]
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43) ~[na:1.8.0_372]
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223) ~[na:1.8.0_372]
	at sun.nio.ch.IOUtil.read(IOUtil.java:192) ~[na:1.8.0_372]
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:379) ~[na:1.8.0_372]
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:258) ~[netty-buffer-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132) ~[netty-buffer-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357) ~[netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151) ~[netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:722) [netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:658) [netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:584) [netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:496) [netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:995) [netty-common-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-common-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-common-4.1.77.Final.jar:4.1.77.Final]
	at java.lang.Thread.run(Thread.java:750) [na:1.8.0_372]

2023-11-24 21:30:08.907  INFO 1188 --- [lettuce-nioEventLoop-4-2] io.lettuce.core.protocol.CommandHandler  : null Unexpected exception during request: java.io.IOException: 远程主机强迫关闭了一个现有的连接。

java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method) ~[na:1.8.0_372]
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43) ~[na:1.8.0_372]
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223) ~[na:1.8.0_372]
	at sun.nio.ch.IOUtil.read(IOUtil.java:192) ~[na:1.8.0_372]
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:379) ~[na:1.8.0_372]
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:258) ~[netty-buffer-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132) ~[netty-buffer-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357) ~[netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151) ~[netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:722) [netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:658) [netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:584) [netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:496) [netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:995) [netty-common-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-common-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-common-4.1.77.Final.jar:4.1.77.Final]
	at java.lang.Thread.run(Thread.java:750) [na:1.8.0_372]

2023-11-24 21:30:08.907  INFO 1188 --- [lettuce-nioEventLoop-4-1] io.lettuce.core.protocol.CommandHandler  : null Unexpected exception during request: java.io.IOException: 远程主机强迫关闭了一个现有的连接。

java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method) ~[na:1.8.0_372]
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43) ~[na:1.8.0_372]
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223) ~[na:1.8.0_372]
	at sun.nio.ch.IOUtil.read(IOUtil.java:192) ~[na:1.8.0_372]
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:379) ~[na:1.8.0_372]
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:258) ~[netty-buffer-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132) ~[netty-buffer-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357) ~[netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151) ~[netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:722) [netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:658) [netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:584) [netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:496) [netty-transport-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:995) [netty-common-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-common-4.1.77.Final.jar:4.1.77.Final]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-common-4.1.77.Final.jar:4.1.77.Final]
	at java.lang.Thread.run(Thread.java:750) [na:1.8.0_372]

2023-11-24 21:30:09.025  INFO 8292 --- [lettuce-eventExecutorLoop-1-6] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was /127.0.0.1:6379
2023-11-24 21:30:09.025  INFO 8292 --- [lettuce-eventExecutorLoop-1-8] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was /127.0.0.1:6379
2023-11-24 21:30:09.025  INFO 8292 --- [lettuce-eventExecutorLoop-1-7] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was /127.0.0.1:6379
2023-11-24 21:30:09.038  INFO 1188 --- [lettuce-eventExecutorLoop-1-7] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was /127.0.0.1:6379
2023-11-24 21:30:09.038  INFO 1188 --- [lettuce-eventExecutorLoop-1-5] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was /127.0.0.1:6379
2023-11-24 21:30:09.038  INFO 1188 --- [lettuce-eventExecutorLoop-1-6] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was /127.0.0.1:6379
2023-11-24 21:30:11.076  WARN 8292 --- [lettuce-nioEventLoop-4-4] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:11.076  WARN 8292 --- [lettuce-nioEventLoop-4-5] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:11.091  WARN 8292 --- [lettuce-nioEventLoop-4-6] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:11.092  WARN 1188 --- [lettuce-nioEventLoop-4-4] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:11.092  WARN 1188 --- [lettuce-nioEventLoop-4-5] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:11.092  WARN 1188 --- [lettuce-nioEventLoop-4-6] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:15.335  INFO 1188 --- [lettuce-eventExecutorLoop-1-5] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:15.335  INFO 8292 --- [lettuce-eventExecutorLoop-1-8] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:15.335  INFO 1188 --- [lettuce-eventExecutorLoop-1-7] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:15.335  INFO 8292 --- [lettuce-eventExecutorLoop-1-6] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:15.335  INFO 8292 --- [lettuce-eventExecutorLoop-1-7] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:15.335  INFO 1188 --- [lettuce-eventExecutorLoop-1-6] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:17.384  WARN 1188 --- [lettuce-nioEventLoop-4-3] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:17.384  WARN 8292 --- [lettuce-nioEventLoop-4-2] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:17.384  WARN 1188 --- [lettuce-nioEventLoop-4-1] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:17.384  WARN 8292 --- [lettuce-nioEventLoop-4-1] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:17.384  WARN 1188 --- [lettuce-nioEventLoop-4-2] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:17.384  WARN 8292 --- [lettuce-nioEventLoop-4-3] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:21.720  INFO 8292 --- [lettuce-eventExecutorLoop-1-12] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:21.720  INFO 8292 --- [lettuce-eventExecutorLoop-1-1] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:21.720  INFO 8292 --- [lettuce-eventExecutorLoop-1-2] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:21.736  INFO 1188 --- [lettuce-eventExecutorLoop-1-1] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:21.736  INFO 1188 --- [lettuce-eventExecutorLoop-1-11] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:21.736  INFO 1188 --- [lettuce-eventExecutorLoop-1-12] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:23.751  WARN 8292 --- [lettuce-nioEventLoop-4-12] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:23.751  WARN 8292 --- [lettuce-nioEventLoop-4-10] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:23.751  WARN 8292 --- [lettuce-nioEventLoop-4-11] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:23.766  WARN 1188 --- [lettuce-nioEventLoop-4-10] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:23.766  WARN 1188 --- [lettuce-nioEventLoop-4-11] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:23.766  WARN 1188 --- [lettuce-nioEventLoop-4-12] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:28.819  INFO 8292 --- [lettuce-eventExecutorLoop-1-8] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:28.819  INFO 8292 --- [lettuce-eventExecutorLoop-1-6] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:28.819  INFO 8292 --- [lettuce-eventExecutorLoop-1-7] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:28.832  INFO 1188 --- [lettuce-eventExecutorLoop-1-7] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:28.832  INFO 1188 --- [lettuce-eventExecutorLoop-1-6] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:28.832  INFO 1188 --- [lettuce-eventExecutorLoop-1-5] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:30.846  WARN 8292 --- [lettuce-nioEventLoop-4-8] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:30.846  WARN 8292 --- [lettuce-nioEventLoop-4-7] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:30.876  WARN 8292 --- [lettuce-nioEventLoop-4-9] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:30.876  WARN 1188 --- [lettuce-nioEventLoop-4-8] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:30.891  WARN 1188 --- [lettuce-nioEventLoop-4-9] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:30.891  WARN 1188 --- [lettuce-nioEventLoop-4-7] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:36.020  INFO 8292 --- [lettuce-eventExecutorLoop-1-9] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:36.020  INFO 8292 --- [lettuce-eventExecutorLoop-1-11] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:36.020  INFO 8292 --- [lettuce-eventExecutorLoop-1-10] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:36.035  INFO 1188 --- [lettuce-eventExecutorLoop-1-10] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:36.035  INFO 1188 --- [lettuce-eventExecutorLoop-1-9] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:36.035  INFO 1188 --- [lettuce-eventExecutorLoop-1-8] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:38.061  WARN 8292 --- [lettuce-nioEventLoop-4-3] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:38.061  WARN 8292 --- [lettuce-nioEventLoop-4-2] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:38.061  WARN 8292 --- [lettuce-nioEventLoop-4-1] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:38.076  WARN 1188 --- [lettuce-nioEventLoop-4-2] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:38.076  WARN 1188 --- [lettuce-nioEventLoop-4-1] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:38.076  WARN 1188 --- [lettuce-nioEventLoop-4-3] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:42.220  INFO 8292 --- [lettuce-eventExecutorLoop-1-6] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:42.220  INFO 8292 --- [lettuce-eventExecutorLoop-1-7] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:42.220  INFO 8292 --- [lettuce-eventExecutorLoop-1-8] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:42.229  INFO 1188 --- [lettuce-eventExecutorLoop-1-6] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:42.229  INFO 1188 --- [lettuce-eventExecutorLoop-1-5] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:42.229  INFO 1188 --- [lettuce-eventExecutorLoop-1-7] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:44.265  WARN 8292 --- [lettuce-nioEventLoop-4-5] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:44.265  WARN 8292 --- [lettuce-nioEventLoop-4-4] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:44.265  WARN 8292 --- [lettuce-nioEventLoop-4-6] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:44.280  WARN 1188 --- [lettuce-nioEventLoop-4-4] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:44.280  WARN 1188 --- [lettuce-nioEventLoop-4-5] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:44.280  WARN 1188 --- [lettuce-nioEventLoop-4-6] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:52.527  INFO 8292 --- [lettuce-eventExecutorLoop-1-12] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:52.527  INFO 8292 --- [lettuce-eventExecutorLoop-1-1] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:52.527  INFO 8292 --- [lettuce-eventExecutorLoop-1-2] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:52.529  INFO 1188 --- [lettuce-eventExecutorLoop-1-1] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:52.529  INFO 1188 --- [lettuce-eventExecutorLoop-1-11] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:52.529  INFO 1188 --- [lettuce-eventExecutorLoop-1-12] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:30:54.568  WARN 8292 --- [lettuce-nioEventLoop-4-8] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:54.568  WARN 1188 --- [lettuce-nioEventLoop-4-9] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:54.568  WARN 8292 --- [lettuce-nioEventLoop-4-9] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:54.568  WARN 1188 --- [lettuce-nioEventLoop-4-7] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:54.568  WARN 8292 --- [lettuce-nioEventLoop-4-7] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:30:54.568  WARN 1188 --- [lettuce-nioEventLoop-4-8] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:31:05.027 ERROR 8292 --- [pool-4-thread-1] o.s.s.s.TaskUtils$LoggingErrorHandler    : Unexpected error occurred in scheduled task

org.springframework.dao.QueryTimeoutException: Redis command timed out; nested exception is io.lettuce.core.RedisCommandTimeoutException: Command timed out after 5 second(s)
	at org.springframework.data.redis.connection.lettuce.LettuceExceptionConverter.convert(LettuceExceptionConverter.java:70) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.connection.lettuce.LettuceExceptionConverter.convert(LettuceExceptionConverter.java:41) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.PassThroughExceptionTranslationStrategy.translate(PassThroughExceptionTranslationStrategy.java:44) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.FallbackExceptionTranslationStrategy.translate(FallbackExceptionTranslationStrategy.java:42) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.connection.lettuce.LettuceConnection.convertLettuceAccessException(LettuceConnection.java:272) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.connection.lettuce.LettuceConnection.await(LettuceConnection.java:1063) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.connection.lettuce.LettuceConnection.lambda$doInvoke$4(LettuceConnection.java:920) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.connection.lettuce.LettuceInvoker$Synchronizer.invoke(LettuceInvoker.java:665) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.connection.lettuce.LettuceInvoker.just(LettuceInvoker.java:94) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.connection.lettuce.LettuceSetCommands.sMembers(LettuceSetCommands.java:163) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.connection.DefaultedRedisConnection.sMembers(DefaultedRedisConnection.java:866) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.core.DefaultSetOperations.lambda$members$11(DefaultSetOperations.java:244) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.core.RedisTemplate.execute(RedisTemplate.java:224) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.core.RedisTemplate.execute(RedisTemplate.java:191) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.core.AbstractOperations.execute(AbstractOperations.java:97) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.core.DefaultSetOperations.members(DefaultSetOperations.java:244) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.core.DefaultBoundSetOperations.members(DefaultBoundSetOperations.java:162) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.session.data.redis.RedisSessionExpirationPolicy.cleanExpiredSessions(RedisSessionExpirationPolicy.java:132) ~[spring-session-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.session.data.redis.RedisIndexedSessionRepository.cleanupExpiredSessions(RedisIndexedSessionRepository.java:424) ~[spring-session-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54) ~[spring-context-5.3.20.jar:5.3.20]
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:95) [spring-context-5.3.20.jar:5.3.20]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [na:1.8.0_372]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_372]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [na:1.8.0_372]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [na:1.8.0_372]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [na:1.8.0_372]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [na:1.8.0_372]
	at java.lang.Thread.run(Thread.java:750) [na:1.8.0_372]
Caused by: io.lettuce.core.RedisCommandTimeoutException: Command timed out after 5 second(s)
	at io.lettuce.core.internal.ExceptionFactory.createTimeoutException(ExceptionFactory.java:59) ~[lettuce-core-6.1.8.RELEASE.jar:6.1.8.RELEASE]
	at io.lettuce.core.internal.Futures.awaitOrCancel(Futures.java:246) ~[lettuce-core-6.1.8.RELEASE.jar:6.1.8.RELEASE]
	at io.lettuce.core.LettuceFutures.awaitOrCancel(LettuceFutures.java:74) ~[lettuce-core-6.1.8.RELEASE.jar:6.1.8.RELEASE]
	at org.springframework.data.redis.connection.lettuce.LettuceConnection.await(LettuceConnection.java:1061) ~[spring-data-redis-2.7.0.jar:2.7.0]
	... 22 common frames omitted

2023-11-24 21:31:05.034 ERROR 1188 --- [pool-3-thread-1] o.s.s.s.TaskUtils$LoggingErrorHandler    : Unexpected error occurred in scheduled task

org.springframework.dao.QueryTimeoutException: Redis command timed out; nested exception is io.lettuce.core.RedisCommandTimeoutException: Command timed out after 5 second(s)
	at org.springframework.data.redis.connection.lettuce.LettuceExceptionConverter.convert(LettuceExceptionConverter.java:70) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.connection.lettuce.LettuceExceptionConverter.convert(LettuceExceptionConverter.java:41) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.PassThroughExceptionTranslationStrategy.translate(PassThroughExceptionTranslationStrategy.java:44) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.FallbackExceptionTranslationStrategy.translate(FallbackExceptionTranslationStrategy.java:42) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.connection.lettuce.LettuceConnection.convertLettuceAccessException(LettuceConnection.java:272) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.connection.lettuce.LettuceConnection.await(LettuceConnection.java:1063) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.connection.lettuce.LettuceConnection.lambda$doInvoke$4(LettuceConnection.java:920) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.connection.lettuce.LettuceInvoker$Synchronizer.invoke(LettuceInvoker.java:665) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.connection.lettuce.LettuceInvoker.just(LettuceInvoker.java:94) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.connection.lettuce.LettuceSetCommands.sMembers(LettuceSetCommands.java:163) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.connection.DefaultedRedisConnection.sMembers(DefaultedRedisConnection.java:866) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.core.DefaultSetOperations.lambda$members$11(DefaultSetOperations.java:244) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.core.RedisTemplate.execute(RedisTemplate.java:224) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.core.RedisTemplate.execute(RedisTemplate.java:191) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.core.AbstractOperations.execute(AbstractOperations.java:97) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.core.DefaultSetOperations.members(DefaultSetOperations.java:244) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.data.redis.core.DefaultBoundSetOperations.members(DefaultBoundSetOperations.java:162) ~[spring-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.session.data.redis.RedisSessionExpirationPolicy.cleanExpiredSessions(RedisSessionExpirationPolicy.java:132) ~[spring-session-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.session.data.redis.RedisIndexedSessionRepository.cleanupExpiredSessions(RedisIndexedSessionRepository.java:424) ~[spring-session-data-redis-2.7.0.jar:2.7.0]
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54) ~[spring-context-5.3.20.jar:5.3.20]
	at org.springframework.scheduling.concurrent.ReschedulingRunnable.run(ReschedulingRunnable.java:95) [spring-context-5.3.20.jar:5.3.20]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [na:1.8.0_372]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_372]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [na:1.8.0_372]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [na:1.8.0_372]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [na:1.8.0_372]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [na:1.8.0_372]
	at java.lang.Thread.run(Thread.java:750) [na:1.8.0_372]
Caused by: io.lettuce.core.RedisCommandTimeoutException: Command timed out after 5 second(s)
	at io.lettuce.core.internal.ExceptionFactory.createTimeoutException(ExceptionFactory.java:59) ~[lettuce-core-6.1.8.RELEASE.jar:6.1.8.RELEASE]
	at io.lettuce.core.internal.Futures.awaitOrCancel(Futures.java:246) ~[lettuce-core-6.1.8.RELEASE.jar:6.1.8.RELEASE]
	at io.lettuce.core.LettuceFutures.awaitOrCancel(LettuceFutures.java:74) ~[lettuce-core-6.1.8.RELEASE.jar:6.1.8.RELEASE]
	at org.springframework.data.redis.connection.lettuce.LettuceConnection.await(LettuceConnection.java:1061) ~[spring-data-redis-2.7.0.jar:2.7.0]
	... 22 common frames omitted

2023-11-24 21:31:11.019  INFO 8292 --- [lettuce-eventExecutorLoop-1-8] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:31:11.019  INFO 8292 --- [lettuce-eventExecutorLoop-1-9] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:31:11.019  INFO 8292 --- [lettuce-eventExecutorLoop-1-7] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:31:11.032  INFO 1188 --- [lettuce-eventExecutorLoop-1-7] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:31:11.032  INFO 1188 --- [lettuce-eventExecutorLoop-1-6] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:31:11.032  INFO 1188 --- [lettuce-eventExecutorLoop-1-8] i.l.core.protocol.ConnectionWatchdog     : Reconnecting, last destination was 127.0.0.1:6379
2023-11-24 21:31:13.067  WARN 8292 --- [lettuce-nioEventLoop-4-12] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:31:13.067  WARN 8292 --- [lettuce-nioEventLoop-4-10] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:31:13.067  WARN 8292 --- [lettuce-nioEventLoop-4-11] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:31:13.082  WARN 1188 --- [lettuce-nioEventLoop-4-11] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:31:13.082  WARN 1188 --- [lettuce-nioEventLoop-4-12] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
2023-11-24 21:31:13.093  WARN 1188 --- [lettuce-nioEventLoop-4-10] i.l.core.protocol.ConnectionWatchdog     : Cannot reconnect to [127.0.0.1:6379]: Connection refused: no further information: /127.0.0.1:6379
